extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -676,14 +723,16 @@ public static JsonArray getJsonResponse(final Page page, final Localization loca
         return JsonUtils.toJsonArray(getValidJsonResponseBody(response));
     }
 
-    public static JsonBuilder<JsonObject> prepareJsonBuilder()
+    public static JsonBuilder<JsonObject> prepareJsonBuilder(final String contentCountry)
             throws IOException, ExtractionException {
         // @formatter:off
         return JsonObject.builder()
                 .object("context")
                     .object("client")
                         .value("clientName", "1")
                         .value("clientVersion", getClientVersion())
+                        .value("hl", "en-GB")
```

Comment:
It should be extractor localization. 

(And it won't change anything right now as for youtube the only allowed language is en-GB but that's the right way to do)

Reply:
If I write the value `en` for the header `hl` or if I don't write this header, the CCLienced test will fail because it's the american English (license, not license)

Reply:
The tests by default should be using https://github.com/TeamNewPipe/NewPipeExtractor/blob/55e0f8e725e45f06513d43875fee6644ea9a8066/extractor/src/main/java/org/schabi/newpipe/extractor/localization/Localization.java#L11 🤔

Reply:
Yes, but it's a requirement for the YouTube Innertube API in order to have the britannic English as the language returned; otherwise you will got the amercian English.

Reply:
Cant you overwrite the Localization for that single test? I agree with B0pol that it should be the extractor localization

Reply:
By extractor localization I meant NewPipe.getPreferredLocalization(). It will be en-GB anyway because we only allow en-GB for YouTube ( https://github.com/TeamNewPipe/NewPipeExtractor/blob/dev/extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeService.java#L189 )

Reply:
Ok, thank you